comprl/server/game_result.py

```diff
@@ -0,0 +1,60 @@
+"""Class to store the result of a game"""
+
+from datetime import datetime
+from typing import Optional
+from enum import Enum
+from uuid import UUID
+
+GameEndState = Enum("GameEndState", ["WIN", "DRAW", "DISCONNECTED"])
+
+
+class GameResult:
```

Comment:
I looked at was a [dataclass](https://docs.python.org/3/library/dataclasses.html) can do and it does not exactly  match what I do in the GameResult class. This class not only stores data bur also reformats it to match the layout of the database. 
I know that you could put the same code in the database class, but that makes this class less readable, and I think that is worse than this class less readable because this class is quite small.
Another possibility would be to put this code into the game class, but then it is possible to insert data into the database that doesn't make sense.
So, I think it is best to leave it as it is and don't use a dataclass.